src/labels/CustomLabels.labels

```diff
@@ -671,6 +671,14 @@
         <shortDescription>stgAfflDeleteProgramEnrollment</shortDescription>
         <value>Status Specified for Affiliations Not Deleted:</value>
     </labels>
+    <labels>
+        <fullName>stgAfflNotInserted</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgAfflNotInserted</shortDescription>
+        <value>There is already an item in the &apos;Affl Mapping&apos; Custom settings with the same name as this Account record type name, please add a different record type.</value>
```

Comment:
I don't think the part "please add a different record type" is correct. We don't really want them to add a different record type. We have to explain to them that an entry with the same record type name is not allowed, but that if they need it for translations they should modify the existing one in the Setup menu and replace the field label with the field api name (keeping the same record type name). Please explain it to Julie and ask her if she could massage the text. You may need to give her a quick demo...

Reply:
Thanks for the suggestion Carlos, I am working with Julie on this, can you please review the rest of the PR meanwhile?